Assets/MRTK/SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs

```diff
@@ -191,9 +214,19 @@ public void SetFollowMeBehavior(bool follow)
                     visualizationObject.SetActive(follow);
                 }
 
-                if (interactableObject != null)
+                if (ButtonBehavior != FollowMeBehaviorToInteractablesToggleState.Manual)
```

Comment:
I think it would be better to do a switch statement here for understandability, something like the following

```suggestion
                switch (ButtonBehavior):
                case FollowMeBehaviorToInteractablesToggleState.ToggledWhenFollowing:
                    interactableObject.IsToggled = follow;
                case FollowMeBehaviorToInteractablesToggleState.UntoggledWhenFollowing:
                    interactableObject.IsToggled = !follow;
                    break;
                case FollowMeBehaviorToInteractablesToggleState.Manual:
                    break;
                
                
```

Reply:
This is exactly what I wanted to do at first. But then I was afraid to make this many changes 😀Let me quickly fix and update the PR.